Renegade X Launcher/LaunchTools.cs

```diff
@@ -64,6 +65,14 @@ public override string GetProcessArguments ()
             return arguments;
         }
 
+        public override string GetProcessPath()
+        {
+            if (Use64bit)
+                return GameInstallation.GetRootPath() + "Binaries\\Win64\\UDK.exe";

```

Comment:
It seems like we could use udklift.exe here instead (which is what GetProcessPath currently does); that way if someone checks "64 Bit Client" on a system that can't support it, it should just use the 32 bit client anyway.